modular_citadel/code/game/objects/items/stunsword.dm

```diff
@@ -0,0 +1,31 @@
+/obj/item/melee/baton/stunsword
+	name = "stunsword"
+	desc = "not actually sharp, this sword is functionally identical to a stunbaton"
+	icon = 'modular_citadel/icons/obj/stunsword.dmi'
+	icon_state = "stunsword"
+	item_state = "sword"
+	lefthand_file = 'modular_citadel/icons/mob/inhands/stunsword_left.dmi'
+	righthand_file = 'modular_citadel/icons/mob/inhands/stunsword_right.dmi'
+
+/obj/item/device/ssword_kit
+	name = "stunsword kit"
+	desc = "a modkit for making a stunbaton into a stunsword"
+	icon = 'icons/obj/custom.dmi' //woop woop placeholder icon
+	icon_state = "cebu"
+	var/product = /obj/item/melee/baton/stunsword //what it makes
+	var/list/fromitem = list(/obj/item/melee/baton, /obj/item/melee/baton/loaded) //what it needs
+	afterattack(obj/O, mob/user as mob)
+		if(istype(O, product))
+			user << "<span class='warning'>[O] is already modified!"
```

Comment:
Never use <<.

Use to_chat(user,"message")

Reply:
Derp, I made that change on lines 29 and 31, but I missed this one because it never came up in testing